src/components/DataTable/index.js

```diff
@@ -103,7 +103,7 @@ export function DataTable ({
 							rowClassName={customRowClassName}
 						/>}
 
-						{isLoading && contentLoader && <CellContentLoader cols={headers.length} rows={limit ?? 5} /> }
+						{isLoading && contentLoader && <CellContentLoader cols={headers.length} rows={limit ?? 5} foregroundColor={theme === "dark" ? '#788dcb' : '#fff'} /> }
```

Comment:
@Aringoaway Is this change necessary? From the code I see if you dont specify foregroundColor, then the content loader will have the default and the default in content loader is the same as here

Reply:
@Pe5h4 fixed it